nix shell SCK implicit declared error

```
[nix-shell:~/workspace/Ulticores]$ make BOARD=tinyfpgabx
echo '`define tinyfpgabx\n`include "./boards/tinyfpgabx/tinyfpgabx.v"' > ./src/generated/board.v
icepll -i 16 -o 64 -m -n spi_pll -f ./src/generated/spi_pll.v

F_PLLIN:    16.000 MHz (given)
F_PLLOUT:   64.000 MHz (requested)
F_PLLOUT:   64.000 MHz (achieved)

FEEDBACK: SIMPLE
F_PFD:   16.000 MHz
F_VCO: 1024.000 MHz

DIVR:  0 (4'b0000)
DIVF: 63 (7'b0111111)
DIVQ:  4 (3'b100)

FILTER_RANGE: 1 (3'b001)

PLL configuration written to: ./src/generated/spi_pll.v
yosys -ql ./logs/tinyfpgabx_yosys.log -p 'synth_ice40 -top top -abc9 -dsp -blif ./build/tinyfpgabx.blif -json ./build/tinyfpgabx.json' ./src/top.v
Warning: Yosys has only limited support for tri-state logic at the moment. (./src/spi.v:30)
./src/top.v:71: ERROR: Identifier `\SCK' is implicitly declared and `default_nettype is set to none.
make: *** [Makefile:34: build/tinyfpgabx.bit] Error 1

[nix-shell:~/workspace/Ulticores]$ which yosys
/nix/store/s8sza410xc34ssmrx27jlmpd9gmwsj0r-yosys-2020.08.22/bin/yosys
```

--- a/Makefile
+++ b/Makefile
@@ -29,7 +29,7 @@ all: $(BUILD).bit
 
 $(BUILD).bit:
 # set board define for Verilog and include the board specific verilog file
-	echo '`define $(BOARD)\n`include "./boards/$(BOARD)/$(BOARD).v"' > $(GENERATEDDIR)board.v
+	printf '`define $(BOARD)\n`include "./boards/$(BOARD)/$(BOARD).v"' > $(GENERATEDDIR)board.v
 ifeq ($(ARCH), ice40)
 	icepll -i $(FREQ) -o $(SPIFREQ) -m -n spi_pll -f $(GENERATEDDIR)spi_pll.v
 	yosys -ql ./logs/$(BOARD)_yosys.log -p 'synth_ice40 -top $(PROJ) -abc9 -dsp -blif $(BUILD).blif -json $(BUILD).json' $(TOP)
@@ -51,7 +51,7 @@ clean:
 	rm -f $(BUILD).blif $(BUILD).asc $(BUILD).rpt  $(BUILD).json $(BUILD).bin $(BUILD).bit ./src/generated/*.v
 
 formal:
-	echo '`define $(BOARD)\n`include "./boards/$(BOARD)/$(BOARD).v"' > $(GENERATEDDIR)board.v
+	printf '`define $(BOARD)\n`include "./boards/$(BOARD)/$(BOARD).v"' > $(GENERATEDDIR)board.v
 	sby -f symbiyosys.sby
 
 lint:

